test/dom.test.js

```diff
@@ -605,11 +605,15 @@ testTreeSegue("styles", [
     html: `<div></div>`
   },
   {
-    tree: h("div", { style: { color: "red", fontSize: "1em" } }),
+    tree: h("div", {
+      style: { color: "red", fontSize: "1em", "--foo": "red" }
+    }),
     html: `<div style="color: red; font-size: 1em;"></div>`
   },
   {
-    tree: h("div", { style: { color: "blue", float: "left" } }),
+    tree: h("div", {
+      style: { color: "blue", float: "left", "--foo": "blue" }
+    }),
     html: `<div style="color: blue; float: left;"></div>`
```

Comment:
looks like jsdom does not support custom properties yet =/
```js
test('example', () => {
  const div = document.createElement('div')
  div.style.setProperty('--foo', 'red')
  document.body.appendChild(div)

  console.log(document.body.innerHTML) // => '<div></div>'
  console.log(div.style.getPropertyValue('--foo')) // => undefined

  expect(document.body.innerHTML).toBe('<div style="--foo:red"></div>') // => fail
  expect(div.style.getPropertyValue('--foo')).toBe('red') // => fail
})
```

Reply:
Google Search crawler is based on Chrome 41, which does NOT support CSS Custom Properties: https://twitter.com/ebidel/status/968989651888295936